Assets/Scripts/Nodes/IsDestinationWithinLimits.cs

```diff
@@ -5,16 +5,16 @@
 
 public class IsDestinationWithinLimits : LeafNode
 {
-	Vector3 _destination;
+	private Hashtable _data;
 
 	public override void InitSelf( Hashtable data )
 	{
-		_destination = (Vector3)data["destination"];
+		_data = data;
 	}
 
 	public override NodeStatus TickSelf()
 	{
-		if ( MathUtils.IsWithinInfiniteVerticalCylinders( _destination, LimitsManager.colliders ) )
+		if ( MathUtils.IsWithinInfiniteVerticalCylinders( (Vector3) _data["destination"], LimitsManager.colliders ) )
```

Comment:
I assume this is the fix for the cylinder error that was coming up? What was the issue?